code-challenges.js

```diff
@@ -21,49 +21,109 @@ const fruit4 = "kiwi"
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
-
+// Create a function called boilingPoint that takes in a number
+//if number is less than 212 return below boiling point
+//else if number is euall to 212 return at boiling point 
+//else meaning any other number return above boiling point
+
+const boilingPoint = (num) => {
+    if(num < 212){
+        return "below boiling point"
+    }
+    else if(num === 212){
+        return "at boiling point"
+    }
+    else{
+        return "above boiling point"
+    }
+}
 const temperature1 = 42
 // Expected output: "42 is below boiling point"
-
+console.log(boilingPoint(42))
 const temperature2 = 350
 // Expected output: "350 is above boiling point"
-
+console.log(boilingPoint(350))
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
+console.log(boilingPoint(212))
+
+
+
+
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+//console log padres 1884 concat to combine with 1898
+//length to get the new count 
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+
+
+console.log([padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns).length])
 // Expected output: 9
 
+
+
+
+
+
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
-
+//create a var splitString to convert string to array 
+//create var reverseArray to reverse the new array that was created 
+//create var joinArray  to convert the new reversed array to a string 
+//console log joinArray 
 const currentCohort = "Delta 2023"
+var splitString = currentCohort.split("")
+var reveseArray = splitString.reverse()
+var joinArray = reveseArray.join("")
```

Comment:
Just like how you chained `.concat` and `.length` on the same line in question 2, you can chain all of these built-in methods you are using here on the same line as well.